tests/ops/integration_tests/saas/test_shopify_task.py

```diff
@@ -1,828 +1,68 @@
-            "title",
-            "updated_at",
-            "commentable",
-            "feedburner",
-            "feedburner_location",
-            "created_at",
-            "template_suffix",
-            "tags",
-            "admin_graphql_api_id",
-        ],
-    )
-
-    assert_rows_match(
-        v[f"{dataset_name}:blog_articles"],
-        min_size=1,
-        keys=[
-            "id",
-            "title",
-            "created_at",
-            "body_html",
-            "blog_id",
-            "author",
-            "user_id",
-            "published_at",
-            "updated_at",
-            "summary_html",
-            "template_suffix",
-            "handle",
-            "tags",
-            "admin_graphql_api_id",
-        ],
-    )
-
-    assert_rows_match(
-        v[f"{dataset_name}:blog_article_comments"],
-        min_size=1,
-        keys=[
-            "id",
-            "body",
-            "body_html",
-            "author",
-            "email",
-            "status",
-            "article_id",
-            "blog_id",
-            "created_at",
-            "updated_at",
-            "ip",
-            "user_agent",
-            "published_at",
-        ],
-    )
-
-    temp_masking = CONFIG.execution.masking_strict
-    CONFIG.execution.masking_strict = True
-
-    x = erasure_runner_tester(
-        privacy_request,
-        erasure_policy_complete_mask,
-        graph,
-        [shopify_connection_config],
-        {"email": shopify_erasure_identity_email},
-        get_cached_data_for_erasures(privacy_request.id),
-        db,
-    )
-    if dsr_version == "use_dsr_3_0":
-        assert x == {
-            f"{dataset_name}:customers": 1,
-            f"{dataset_name}:blogs": 0,
-            f"{dataset_name}:customer_orders": 1,
-            f"{dataset_name}:blog_articles": 0,
-            f"{dataset_name}:blog_article_comments": 1,
-            f"{dataset_name}:customer_order_transactions": 0,
+class TestShoppifyConnector:
+
+    def test_connection(self, shopify_runner: ConnectorRunner):
+        shopify_runner.test_connection()
+
+    async def test_access_request(
+        self,
+        shopify_runner: ConnectorRunner,
+        policy: Policy,
+        shopify_identity_email: str,
+        shopify_access_data,
+    ):
+        access_results = await shopify_runner.access_request(
+            access_policy=policy, identities={"email": shopify_identity_email}
+        )
+        key = shopify_runner.dataset_config.fides_key
+
+        ## Assert for customers email
+        for customer in access_results[f"{key}:customers"]:
+            assert customer["email"] == shopify_identity_email
+
+        ## Assert for Orders by Customer
+        orders = access_results[f"{key}:customer_orders"]
+        assert len(orders) == len(shopify_access_data["orders"])
+        for order in orders:
+            assert order["email"] == shopify_identity_email
+
+        ## Assert for blog article comments
```

Comment:
Comments should only start with a single `#`
```suggestion
        # Assert for blog article comments
```